src/testUtils/logging.ts

```diff
@@ -14,14 +14,13 @@ export function makeMockLogging(): MockLogging {
     logs.push(JSON.parse(data));
   });
   const logger = pino({ level: 'debug' }, stream);
-  return { logger, logs };
-}
 
-// eslint-disable-next-line import/no-unused-modules
-export function partialPinoLogger(bindings: { readonly [key: string]: any }): any {
-  return expect.objectContaining({
-    [PinoSymbols.formattersSym]: { bindings },
+  beforeEach(() => {
+    // Clear the logs
+    logs.splice(0, logs.length);
```

Comment:
I accidentally wrote a test that passed because of a log that was recorded in a previous test, so that's why I'm forcing the set of logs to be cleared between tests.